tokio-console/src/conn.rs

```diff
@@ -78,7 +82,23 @@ impl Connection {
                 tokio::time::sleep(backoff).await;
             }
             let try_connect = async {
-                let mut client = InstrumentClient::connect(self.target.clone()).await?;
+                let channel = match self.target.scheme_str() {
+                    None | Some("file") => {
```

Comment:
i'm not sure how i feel about treating URLs without a scheme as UDS...is that the generally accepted convention?

Reply:
I don't think there _is_ a generally accepted convention. Some other software uses `unix:` as the scheme, as in `unix://path/to/socket`. For now I just removed the treatment of `None` as UDS; let me know if you'd prefer some other scheme instead.